### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by sklam**

### CODE REVIEW

File: CHANGE_LOG [hunk 1]
```diff
@@ -1,3 +1,30 @@
+Version 0.29.0
+--------------
+
+This release extends the support of recursive functions to include direct and
+indirect recursion without explicit function type annotations.  See new example
+in `examples/mergesort.py`.  Newly supported numpy features include array
+stacking functions, np.linalg.eig* functions, np.lingla.matrix_power and
```
Comment (stuartarchibald) [inline CHANGE_LOG]: s/lingla/linalg/

Comment (stuartarchibald) [inline CHANGE_LOG]: Also, `np.roots()` was added #2135.

**Comment by seibert** [comment]
Should note that llvmlite 0.14 will be required for this release.

Missing PRs:
* #2115
* #2117 
* #2119 - fix array type when using a `Record` dtype
* #2122 
* #2127

**Comment by seibert** [comment]
Also note that CUDA 8 is supported, but not required.

File: CHANGE_LOG [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+* PR #2128: Allow for vectorizing a jitted function. (thanks to @dhirschfeld)
+* PR #2117: Update examples and make them test-able.
+* PR #2127: Refactor interpreter class and its results.
+
+Fixes:
+
+* PR #2149: Workaround MSVC9.0 SP1 fmod bug kb982107.
+* PR #2145, Issue #2009: Fixes kwargs for jitclass ``__init__`` method.
+* PR #2150: Fix slowdown in objmode fallback.
+* PR #2050, Issue #1258: Fix liveness problem with some generator loops.
```
Comment (ma-ba) [inline CHANGE_LOG]: Typo: #1259 is the issue fixed by PR #2050

Comment (ma-ba) [inline CHANGE_LOG]: Typo: #1259 is the issue fixed by PR #2050

**Pull request merged**